app/styles/ui/history/_expandable-commit-summary.scss

```diff
@@ -108,6 +108,10 @@
           height: auto;
           min-width: 16px;
 
+          .octicon {
+            scale: 75%;
+          }
```

Comment:
This seems needed also in the submodule diff screen:
<img width="960" alt="image" src="https://github.com/desktop/desktop/assets/1083228/722bfe45-1bcc-415f-9e5c-627892cdfc16">

Reply:
> It looks good overall but I'm a bit concerned about the need to scale down some icons… don't they all have the same size as they used to do?

@sergiou87 I think that's exactly the case. I think someone icons used to only come in the 12px and 24px size, and are now available at 16px. Digging into the current app, it looks like the copy icon was only available at 24px, and then it was scaled down to 12px. I think the actual best solution in the future is to save all of the heights in the generated file, and then choose between them with a prop to the octicons component.

Instead of scaling by 75%, I instead modified all of the `.copy-button` styles. I've also used `check-circle-fill` instead of the check icon with a circular element around it. It achieves the same effect at the same size.

Reply:
@sergiou87 I created a branch off of this PR that goes even further. I've adjusted icon generation to output all sizes and the path data. Then adjusted the `Octicons` component and other components to use either a new custom type or the updated generated output.

https://github.com/desktop/desktop/compare/jc-icons-bigger-update?expand=1

I think the work in the new PR should supercede this PR, as we can now target specific sizes for the maximum sharpness. The existing development branch uses a 24px copy icon that is then scaled down to 12px. This new branch enables choosing a specific size and using that, so that we don't default to the new 16px icon that looks blurry when scaled.